### ISSUE TITLE
False positive for `WPS413`

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#927]
# Bug report

## What's wrong

This code produces incorrect results:

```python
class _Callable(object):
    # Callable[[CenterProtocol[T]], T]
    def __getattr__(self, key: str) -> Callable[[object], T1]:
        return operator.attrgetter(key)
```

Output:

```
  32:5     WPS413 Found bad magic module function: __getattr__
  def __getattr__(self, key: str) -> Callable[[object], T1]:
```

## How is that should be

It should work correctly: it is not a module level ` __getattr__`, it is a class-level method.

## System information

I am using `0.12.5`
